OpenAPI/decentralized-iso20022-v1.yaml

```diff
@@ -1,152 +1,395 @@
+                DbtrAgt:
+                  type: object
+                  properties:
+                    FinInstnId:
+                      $ref: '#/components/schemas/FinInstnId'
+                CdtrAgt:
+                  type: object
+                  properties:
+                    FinInstnId:
+                      type: object
+                      description: Unique and unambiguous identification of a financial institution, as assigned under an internationally recognised or proprietary identification scheme.
+                      properties:
+                        BICFI:
+                          type: string
+                          example: BANKANC0XXX
+                Cdtr:
+                  type: object
+                  properties:
+                    Nm:
+                      type: string
+                      example: TEST CORP
+                      description: Name by which a party iand which is usually used to identiparty.
+                      minLength: 1
+                      maxLength: 140  
+                    PstlAdr:
+                      $ref: '#/components/schemas/PstlAdr'
+    PstlAdr:
+      type: object
+      properties:
+        AdrLine:
+          type: string
+          example: 310 Field Road, NY
+          description: Informatiand identifies a specidefined by postal servicefree format text.
+          maxLength: 70             
+    Nm:
+      type: string
+      example: Joe Doe
+      description: Name by which a party iand which is usually used to identiparty.
+      maxLength: 140
+    FinInstnId:
+      type: object
+      description: Unique and unambiguous identification of a financial institution, as assigned under an internationally recognised or proprietary identification scheme.
+      properties:
+        BICFI:
+          type: string
+          example: FOOBARC0XXX
+    BadRequest:
       type: object
       xml:
         attribute: false
         wrapped: false
-        name: RequestPayload
+        name: error
       properties:
-        AppHdr:
-          $ref: '#/components/schemas/AppHdr'
-        Document:
-          $ref: '#/components/schemas/Document'
-    AppHdr:
+        timestamp:
+          type: string
+          example: 2021/05/12 13:56:41
+        status:
+          type: integer
+          example: 400
+        code: 
+          type: string
+          example: BAD REQUEST
+        message:
+          type: string
+          example: The message was not a valid pacs message.
+    Forbidden:
       type: object
+      xml:
+        attribute: false
+        wrapped: false
+        name: error
       properties:
-        something:
+        timestamp:
           type: string
-    Document:
+          example: 2021/05/12 13:56:41
+        status:
+          type: integer
+          example: 403
+        code: 
+          type: string
+          example: FORBIDDEN
+        message:
+          type: string
+          example: a message describing the problem
```

Comment:
I got the impression of the example message for 400 Bad Request that it should be an example that actually shows up in use, but here in 403 Forbidden it sounds like a description of the message that will show up in use. Maybe it should be synchronized so that all examples are one or the other?

Reply:
The problem is that I couldn't figure how what message goes for what error. Could you send me the messages perhaps?

Reply:
An example for 403 is `message.bad-internal-cert`. You can check the controller classes as that's where most of the `ResponseStatusException`s are thrown. The validation service classes also keep some of the lower level error messages for when messages are invalid.

Reply:
Thanks! Found them!